include/drives.h

```diff
@@ -87,10 +87,10 @@ class localDrive : public DOS_Drive {
 	localDrive(const char* startdir, uint16_t _bytes_sector,
 	           uint8_t _sectors_cluster, uint16_t _total_clusters,
 	           uint16_t _free_clusters, uint8_t _mediaid,
-	           bool _always_open_ro_files = false);
+	           bool _readonly, bool _always_open_ro_files = false);
 	bool FileOpen(DOS_File** file, char* name, uint8_t flags) override;
-	virtual FILE* GetSystemFilePtr(const char* const name, const char* const type);
-	virtual bool GetSystemFilename(char* sysName, const char* const dosName);
+	FILE* GetSystemFilePtr(const char* const name, const char* const type);
+	std::string GetSystemFilename(const char* const dosName);
```

Comment:
I'm not saying to refactor the whole file, but it would beg good to fix the naming conventions when we touch a function. So `dos_name` in this case, etc.

Reply:
Renamed